internal/api/health_check.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+var buildTag = ""
+var buildHash = ""
+
+type healthCheckResponse struct {
+	Status      string
+	Version     string
+	ReleaseID   string
+	Notes       []string
+	Description string
+}
+
+func initHealthCheck(apiRouter *mux.Router, context *Context) {
+	addContext := func(handler contextHandlerFunc) *contextHandler {
```

Comment:
I dislike that I had to re-add this here. Maybe it's worth to refactor into it's own method.